biojava-core/src/main/java/org/biojava/nbio/core/sequence/loader/UniprotProxySequenceReader.java

```diff
@@ -638,26 +640,33 @@ public String getOrganismName() {
         	for (Element element : dbreferenceElementList) {
         		String type = element.getAttribute("type");
         		String id = element.getAttribute("id");
-        		ArrayList<DBReferenceInfo> idlist = databaseReferencesHashMap.get(type);
-        		if (idlist == null) {
-        			idlist = new ArrayList<DBReferenceInfo>();
-        			databaseReferencesHashMap.put(type, idlist);
-        		}
-        		DBReferenceInfo dbreferenceInfo = new DBReferenceInfo(type, id);
+        		//ArrayList<DBReferenceInfo> idlist = databaseReferencesHashMap.get(type);
+        		//if (idlist == null) {
+        		//	idlist = new ArrayList<DBReferenceInfo>();
+        		//	databaseReferencesHashMap.put(type, idlist);
+        		//}
+        		//DBReferenceInfo dbreferenceInfo = new DBReferenceInfo(type, id);
         		ArrayList<Element> propertyElementList = XMLHelper.selectElements(element, "property");
         		for (Element propertyElement : propertyElementList) {
         			String propertyType = propertyElement.getAttribute("type");
         			String propertyValue = propertyElement.getAttribute("value");
-        			dbreferenceInfo.addProperty(propertyType, propertyValue);
+        			//dbreferenceInfo.addProperty(propertyType, propertyValue);
+        			
+        			System.out.println("here comes a dbxref, with: type: "+type+", id: "+id+", propertyType: "+propertyType+", propertyValue: "+propertyValue);
```

Comment:
While addressing the above issues, I encounter another issue, which will take a bit, so patience please.